package.json

```diff
@@ -121,6 +121,7 @@
     "autofix-tests": "node test/scripts/integration.js",
     "coverage": "unit-coverage run -p common",
     "coverage-html": "unit-coverage run -p common -r html -o coverage.html",
+    "check-my-coverage": "npm run coverage-html && open coverage.html",
```

Comment:
True. Wouldn't work on windows. We don't get a lot of tickets where we tell users about coverage-html though. So it might not be worthwhile.